pkg/operator/ceph/file/filesystem_test.go

```diff
@@ -191,6 +237,19 @@ func TestCreateNopoolFilesystem(t *testing.T) {
 	assert.Nil(t, err)
 	validateStart(t, context, fs)
 }
+
+func equals(first, second []string) bool {
```

Comment:
Suggest renaming to `argsEqual` for easier readability.

Reply:
Why not use `reflect.DeepEqual()`?

Reply:
Done